S/SCIP/build_tarballs.jl

```diff
@@ -30,10 +37,12 @@ cmake -DCMAKE_INSTALL_PREFIX=$prefix\
   -DGCG=0\
   -DUG=0\
   -DAMPL=0\
-  -DBOOST=off\
+  -DBOOST=ON\
   -DSYM=bliss\
   -DTPI=tny\
-  -DIPOPT_DIR=${prefix} -DIPOPT_LIBRARIES=${libdir} ..
+  -DIPOPT_DIR=${prefix} -DIPOPT_LIBRARIES=${libdir} \
+  -DBLISS_INCLUDE_DIR=${includedir} -DBLISS_LIBRARY=bliss \
```

Comment:
A matter of taste perhaps, but I would suggest to list each argument on its own line; that makes it easier to see what is there (I tend to overlook the ones coming later in the line *sigh*).
```suggestion
  -DIPOPT_DIR=${prefix} \
  -DIPOPT_LIBRARIES=${libdir} \
  -DBLISS_INCLUDE_DIR=${includedir} \
  -DBLISS_LIBRARY=bliss \
```